flink-autoscaler-plugin-jdbc/src/main/java/org/apache/flink/autoscaler/jdbc/event/JdbcAutoScalerEventHandler.java

```diff
@@ -104,6 +134,43 @@ public void handleScalingEvent(
         }
     }
 
+    @Override
+    public void close() {
+        if (Objects.nonNull(scheduledEventHandlerCleaner)
+                && !scheduledEventHandlerCleaner.isShutdown()) {
+            scheduledEventHandlerCleaner.shutdownNow();
+        }
+    }
+
+    @VisibleForTesting
+    void cleanExpiredEvents() {
+        try {
+            JdbcEventInteractor.ExpiredEventsResult expiredResult =
+                    jdbcEventInteractor.queryExpiredEventsAndMaxId(eventHandlerTtl);
+            if (Objects.isNull(expiredResult)) {
+                log.warn("No expired event handlers queried at {}", new Date());
+                return;
+            }
+            var batch = 4098;
+            var startId = expiredResult.minId;
+            var endId = Math.min(startId + batch, expiredResult.maxId);
+            while (startId <= expiredResult.maxId) {
+                jdbcEventInteractor.deleteExpiredEventsByIdRange(startId, endId);
```

Comment:
The `limit` is not supported in postgres delete clause.
So, I try to use the current way, but I found there's no sequence related to the id of the corresponding table for auto incremental primary key in the postgresql ddl.

Should we add it if the current way was accepted?